project/Build.scala

```diff
@@ -1144,9 +1163,20 @@ object Build extends sbt.Build {
             val outFile = dir / "SourceMapTest.scala"
             IO.write(outFile, replaced.replace("0/*<testCount>*/", i.toString))
             Seq(outFile)
-          }
+          },
+
+        testFrameworks += new TestFramework("com.novocode.junit.JUnitFramework"),
```

Comment:
Is this line still necessary? I suspect not, because [sbt adds it by default](https://github.com/sbt/sbt/blob/2c4583e31244af308018baecd23c294ba6158583/main/src/main/scala/sbt/Defaults.scala#L415-L418).